### DISCUSSION TITLE
Darker theme doesn't invert colors in the document to fully black, however Light theme does. Why?

### DISCUSSION THREAD

**Discussion by student13371** [sumatrapdfreader/sumatrapdf#4668] [Q&A]
**Action number 1:** 
SumatraPDF > Settings > Theme > Light
Press keyboard key I to invert colors in the document.

**Outcome of Action number 1:**
Page background is fully black.
Please see the image below for illustration (note fully black background):

![sumatrapdf white theme](https://github.com/user-attachments/assets/1aee1436-0277-4b0c-9e39-d6c356c0a7f5)

**Action number 2:** 
SumatraPDF > Settings > Theme > Darker
Press keyboard key I to invert colors in the document.

**Outcome of Action number 1:**
Page background is NOT fully black, but grey in color.
Please see the image below for illustration (note grey background):

![sumatrapdf darker theme](https://github.com/user-attachments/assets/57ec1396-1508-4c78-a38d-53a0cf55e934)

Is it possible to make Darker theme when inverting colors in the document to show page background fully black the same way as White theme does when colors are inverted?

**Comment by GitHubRulesOK** [comment]
With Darkest in pre-release and inverted it will be as dark as this

![image](https://github.com/user-attachments/assets/38ffec86-6c7d-4ec7-96e0-3409e84d68f1)

At a distance the inversion will affect the text appearance
![image](https://github.com/user-attachments/assets/b93318a4-0721-437b-9bf4-e15093c5933c)

**Reply by student13371** [reply]
Dear GitHubRulesOK,

I have just installed the Latest SumatraPDF pre-release build version 16779 and SumatraPDF > Settings > Theme > Set theme 'Dark' achieves the same fully dark background on inversion as Light theme.

Thank you!
